models/payment_sync.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+import logging
+import requests
+import xml.etree.ElementTree as ElementTree
+from datetime import datetime
+from odoo import api, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentSync(models.Model):
+    _name = "payment.sync"
+
+    @api.model
+    def sync_payments(self):
```

Comment:
the model is already called PaymentSync so calling this method just **sync** **sync_payments** instead of would be enough.